packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -35,24 +78,42 @@ export default class MeetingInfoV2 {
      * Fetches meeting info from the server
      * @param {String} destination one of many different types of destinations to look up info for
      * @param {String} [type] to match up with the destination value
+     * @param {String} password
+     * @param {Object} captchaInfo
+     * @param {String} captchaInfo.code
+     * @param {String} captchaInfo.id
      * @returns {Promise} returns a meeting info object
      * @public
      * @memberof MeetingInfo
      */
-  async fetchMeetingInfo(destination, type = null) {
+  async fetchMeetingInfo(destination, type = null, password = null, captchaInfo = null) {
     const destinationType = await MeetingInfoUtil.getDestinationType({
       destination,
       type,
       webex: this.webex
     });
-    const body = await MeetingInfoUtil.getRequestBody(destinationType);
+    const body = await MeetingInfoUtil.getRequestBody({...destinationType, password, captchaInfo});
 
     return this.webex.request({
       method: HTTP_VERBS.POST,
       service: 'webex-appapi-service',
       resource: 'meetingInfo',
       body
-    });
+    })
+      .catch((err) => {
+        if (err?.statusCode === 403) {
```

Comment:
There are a few error codes and yeah initially I've put them all here, but then I've found the equivalent UCF source code and they don't check the error codes in UCF, only the http code like 403 or 423 so I thought it will be better to align with UCF on this (it means that if wbxappapi add 1 more 403xxxx code for something we won't have to change the SDK code).

Reply:
If all the scenarios are working fine then yes , But i dont trust these guys